### PR TITLE
Build completed for the ustory OM-437ser 

### PR DESCRIPTION
**Pull request by Citronhariom**
Descriptions : Build completed for the   user tory OM-437ser

### CODE REVIEW

**Comment by MHigel** [comment]
@Citronhariom it looks like your commit is missing the new Field Update metadata records that are needed by this Workflow.

File: force-app/main/default/workflows/Opportunity.workflow-meta.xml [hunk 1]
```diff
@@ -447,6 +447,46 @@
         </criteriaItems>
         <triggerType>onCreateOnly</triggerType>
     </rules>
+
+
+   <rules>
+        <fullName>Copy Legal Entity to Invoice address</fullName>
+        <actions>
```
Comment (MHigel) [inline force-app/main/default/workflows/Opportunity.workflow-meta.xml:454]: These Field Updates are missing from the commit.

**Review by MHigel — CHANGES_REQUESTED** [review]

File: force-app/main/default/workflows/Opportunity.workflow-meta.xml (hunk 1 above)
Comment (Citronhariom) [reply force-app/main/default/workflows/Opportunity.workflow-meta.xml:512]: Hi @MHigel ,

Updated the file Please review it once let me know if any changes required from my side.

**Comment by MHigel** [comment]
@Citronhariom there is a merge conflict on this file. Can you resolve it and recommit?

**Comment by Citronhariom** [comment]
Hi @MHigel ,

The conflict has been resolved , Could you please review and let me know if anything required.

**Comment by darcy-old-git** [comment]
Please submit a new PR

**Pull request closed without merging**